.travis.yml

```diff
@@ -1,15 +1,22 @@
+# Forces faster Travis-CI Container Infrastructure
+sudo: false
```

Comment:
Yes, but you can run into issues with mixed `sudo: true` and the default `sudo: false`. Small line addition to ensure less testing issues.